manager/allocator/network.go

```diff
@@ -735,10 +735,10 @@ func (a *Allocator) allocateTask(ctx context.Context, tx store.Tx, t *api.Task)
 	}
 
 	// Update the network allocations and moving to
-	// ALLOCATED state on top of the latest store state.
+	// PENDING state on top of the latest store state.
 	if a.taskAllocateVote(networkVoter, t.ID) {
-		if storeT.Status.State < api.TaskStateAllocated {
-			updateTaskStatus(storeT, api.TaskStateAllocated, "allocated")
+		if storeT.Status.State < api.TaskStatePending {
+			updateTaskStatus(storeT, api.TaskStatePending, "allocated")
```

Comment:
> Maybe define status message as constant. Here it's important to show the difference of "waiting for task to be scheduled" and "blocked by scheduler".

I've changed this to "pending task scheduling" (defined in a constant). Once the scheduler runs, if the task can't be scheduled, it will update the message to explain why.